docs/bestPractises/pullRequest-appengers.md

```diff
@@ -0,0 +1,34 @@
+# Best practices: Pull requests
+This document describes best practices when dealing with pull requests, that were learned within the Appengers team.
+
+## Creation of the pull request
+### Name
+Set the git commit message as the name of the pull request. When merging this is the suggested name for the commit message, which lets you easily confirm the PR instead of setting a proper git commit message. 
+
+#### Examples
+- fix: reflash feature issues
+- feat: add translation for contact and legal notice page 
+
+### Description
+The description is already outlined by our template, which can be found in any of our repositories. Basically it contains all needed information:
```

Comment:
Can you add the template (as MD) to this repository? This way we have a single point to reference to. Also others can learn from it

Reply:
Added as file and linked at the top, so it can be found easier.